brands/amenity/restaurant.json

```
@@ -842,6 +842,15 @@
       "name": "Famous Dave's"
     }
   },
+  "amenity/restaurant|Fatz": {
+    "countryCodes": ["ca", "us"],
```

Took a quick look at their wikipedia and I don't think they operate in Canada?
```suggestion
    "countryCodes": ["us"],
```